utils/jwt_utils.py

```
@@ -0,0 +1,34 @@
+
+        if payload.get("iss") != ISSUER:
+            raise HTTPException(status_code=401, detail="잘못된 토큰 발급자")
+
+        user_id = payload.get("sub")
+        if not user_id:
+            raise HTTPException(status_code=401, detail="payload에 sub 없음")
+
+        return user_id
+
+    except ExpiredSignatureError:
+        raise HTTPException(status_code=401, detail="토큰 만료")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)